test/mediator/auctionmediator.js

```diff
@@ -67,7 +61,7 @@ describe('Pubfood AuctionMediator', function testPubfoodMediator() {
   });
 
   afterEach(function() {
```

Comment:
This `afterEach` is not needed since the concerns are already dealt with in the `beforeEach` so I'd say this should just be removed here and in the other place `afterEach` is being used.